backend/loaders/errors.go

```diff
@@ -0,0 +1,11 @@
+package loaders
+
+import "github.com/ansel1/merry/v2"
+
+const (
+	ErrorCodeNotFound = "not-found"
+)
+
+func errNotFound(id any) error {
+	return merry.Sentinel(ErrorCodeNotFound, merry.WithValue("id", id))
+}
```

Comment:
This is not a sentinel error, because you are re-creating it. A sentinel error is of the style:

```golang
var errX = merry.Sentinel("blah")

func x() error {
   return merry.Wrap(errX, merry.WithValue("id", "blah"))
}

func main() {
  err := x()
  // Then you can do this
  if errors.Is(err, errX) {
    panic("ERROR")
  }
}
```

Reply:
ah ok.
This was reverted already so ill just treat the PR as approved 👍